share/spice/biblio_search/books_on_personality/biblio_search_books_on_personality.js

```diff
@@ -0,0 +1,85 @@
+(function(env) {
+  "use strict";
+
+  env.ddg_spice_biblio_search_books_on_personality = function(api_result) {
+    console.info(api_result);
+    
+    String.prototype.trunc = String.prototype.trunc ||
```

Comment:
We have a Handlebars Helper that can be used instead: `Handlebars.helpers/ellipsis(inputString, 50)`

e.g. https://github.com/duckduckgo/zeroclickinfo-spice/blob/04aaf11a04664c390accb8d5676618eacfa4b950/share/spice/in_theaters/in_theaters.js#L69